Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -153,6 +157,12 @@ The [zone](../../Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.
 
 * `name`: the name of the ceph-object-zone the object store will be in.
 
+## Hosting Settings
+
+The `hosting` settings allows to the host buckets in the object store on a custom DNS name.
+
+* `dnsNames`: a list of DNS names to host buckets on. This provides AWS [virtual-hosted-style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) access to buckets. These names need to valid according RFC-1123. Otherwise it will fail. The multiple hostname support is only available from ceph v18 onwards.
```

Comment:
There are a few pieces of info missing that are critical for users to understand. All the info that is critical for users to understand before using the feature includes:
- this allows virtual-host-style access to buckets (AWS link)
- this further requires that each endpoint has wildcard support (add link)
- there should be no entry that includes the wildcard itself, just the hostname (e.g., s3.mycorp.net)
- do not add hostnames that lack wildcard support (like the default CephObjectStore service)
- DNS hostnames are codified by RFC-1123
- this is supported only for v18+ (there is no need to qualify "multiple hostname support") 